migrations/migrations/20180925215318-user-migration.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
```

Comment:
Remove brackets and return

Reply:
i don't understand this comment

Reply:
If you have this `() => { return something; }` is the same as `() => something`, is only syntax sugar